src/configurationProvider.ts

```diff
@@ -262,3 +238,43 @@ function convertLogLevel(commonLogLevel: string) {
             return "FINE";
     }
 }
+
+interface IMainClassOption {
+    readonly projectName?: string;
+    readonly mainClass?: string;

```

Comment:
Can `mainClass` be empty?

Reply:
as a model, I don't want to forbid mainClass to null